rhel/SPECS/grafana.spec

```diff
@@ -95,7 +97,7 @@ install -d -p %{buildroot}%{_sharedstatedir}/%{repo}
 install -d -p %{buildroot}/var/log/%{repo}
 
 %check
-export GOPATH=$(pwd)/_build:%{gopath}
+export GOPATH="$(pwd)/_build"
 #go test ./pkg/api
 go test ./pkg/bus
```

Comment:
Does it make sense to test only that package? I think we either test all (and that would be `make test-go` or maybe even `make test`: https://github.com/grafana/grafana/blob/master/Makefile) or test nothing.

Reply:
I didn't touch testing part of RPM spec. Let's try adding full tests.

Reply:
"It's not my garbage, it was there before."
https://www.matheus.ro/2017/12/11/clean-code-boy-scout-rule/

Reply:
Acutually, I've planned to clean that out, but only after successful build. :)